### ISSUE TITLE
Update Addon Markdown Forever

### ISSUE THREAD

**Issue by ninhhoang2005** [Andre9642/nvda-markdownForever#18]
Currently, NVDA has the 2025 version, but Addon Mackdown Forever cannot operate due to compatible errors.
You can consider updating the Addon to make all activities happen more smoothly.
best regards
